index.js

```diff
@@ -60,8 +61,30 @@ const sendRequest = (async (method, params) => {
   })
 })
 
+const decodeTransaction = async (transaction_bytecode) => {
+  let decodedTransaction = await sendRequest('decoderawtransaction', [transaction_bytecode])
+  return decodedTransaction
+}
+
+const getTransactionData = async (transaction_hashes) => {
+  const decodedTransactions = []
+  for (let i = 0; i < transaction_hashes.length; i++) {
```

Comment:
When you do not need the index you may prefer using:
`for (const transaction_hash in transaction_hashes)`

Reply:
changed